### ISSUE TITLE
Opening Github links

### ISSUE THREAD

**Issue by drumnkyle** [GitHawkApp/GitHawk#1894]
I love using GitHawk to browse my repos, but I have private repos and getting them into GitHawk is annoying if there is no notification. It would be great to be able to enter a direct link to a repo somewhere in the app so that I can open that repo and bookmark it. Even if it just checked the clipboard on waking from background, that would be suitable.

**Comment by rnystrom** [comment]
I wonder if we could build an "open in GitHawk" extension or something? We won't be able to intercept github.com urls unfortunately.

**Comment by drumnkyle** [comment]
Yeah I know you can’t intercept. I’m talking more about just entering a URL into the app and the app showing the repo. And I was suggesting detecting there is a URL on the clipboard similar to how apps like Instapaper work. But doing an extension could be good too.

**Comment by BasThomas** [comment]
Or both!

**Comment by rnystrom** [comment]
Oh sure! That'd be pretty simple I think.

**Comment by inkoperative** [comment]
#2602
